src/fastertransformer/kernels/BUILD

```diff
@@ -38,6 +38,19 @@ cc_library(
 
 cc_library(
     name = "kernels",
+    hdrs = [
```

Comment:
Are these newly added header files dedicated for rocm ?  If so, please move them into a seperate dir and create a new rocm_kernel_header target.

Reply:
No, they are part of the refactoring effort to split the `kernels/decoder_masked_multihead_attention_utils.h` into multiple smaller files.

Reply:
I see. In that case, starting the filename with underscore is generally not a good idea. Please consider move them into dedicated dir in upcoming changes.